panoramapublic/src/org/labkey/panoramapublic/speclib/LibSourceFile.java

```diff
@@ -81,4 +81,14 @@ public boolean isMaxQuantSearch()
     {
         return (hasIdFile() && getIdFile().endsWith("msms.txt")) || containsScoreType("MAXQUANT SCORE");
     }
+
+    public static String DIANN_REPORT_TSV_PLACEHOLDER = "DIA-NN report file";
+
+    // These are some of the column headers that we expect to see in a DIA-NN report TSV file
+    public static List<String> DIANN_REPORT_EXPECTED_HEADERS = List.of("File.Name", "Run", "Protein.Group", "Protein.Ids", "Protein.Names");
+
+    public boolean isDiannSearch()
+    {
+        return (hasIdFile() && getIdFile().endsWith(".speclib"));
```

Comment:
In some places file extension checks are case-insensitive. Does that matter here?

Reply:
Perhaps.  I've changed it to case-insensitive check.